mmcv/runner/hooks/logger/wandb.py

```diff
@@ -14,13 +16,15 @@ def __init__(self,
                  reset_flag=False,
                  commit=True,
                  by_epoch=True,
-                 with_step=True):
+                 with_step=True,
+                 config_path=None):
```

Comment:
We can add a docstring for the two parameters.

Reply:
The line 19 is just added comma. Can I know how I can make test and docstring?

Reply:
https://github.com/open-mmlab/mmcv/blob/792473c960ed91e334164debc6877a96fdac9558/tests/test_runner/test_hooks.py#L1113-L1130

Reply:
```python
class WandbLoggerHook(LoggerHook):
    """
    Args:
        with_step (bool): xxxx. Default: True.
        config_path (str, optional): xxxx. Default: None.
    """
```

Reply:
I can't find `assert_called_with()` attribute in wandb docs. And `join` is deprecated. Can I know it is the edge of version or working test?

Reply:
[this](https://github.com/wandb/client/issues/1991) is an issue of wandb unit test.

Reply:
We use `MagicMock` to mock the `wandb`. More details about `MagicMock` at https://docs.python.org/3/library/unittest.mock.html#quick-guide

Reply:
My mistake. I just get the code. It is attribute of `mock.py`. I will make a unit test in that format. But `join` is deprecated attribute. It was moved to `finish`. please check it.